scenes/ui/screen/settings_screen/settings_screen.gd

```diff
@@ -23,9 +31,35 @@ func _ready() -> void:
 
 
 func _initialize() -> void:
-	master_settings_slider.title_label.text = Locale.get_ui_label("master")
-	music_settings_slider.title_label.text = Locale.get_ui_label("music")
-	sfx_settings_slider.title_label.text = Locale.get_ui_label("sfx")
+	_apply_effects()
+	_set_ui_labels()
+
+
+func _set_ui_labels() -> void:
+	master_settings_slider.get_title_label().text = Locale.get_ui_label("master")
+	music_settings_slider.get_title_label().text = Locale.get_ui_label("music")
+	sfx_settings_slider.get_title_label().text = Locale.get_ui_label("sfx")
+
+	shake_settings_slider.get_title_label().text = Locale.get_ui_label("shake")
+	typing_settings_slider.get_title_label().text = Locale.get_ui_label("typing")
+
+	display_mode_button.text = "?"
+	display_resolution_button.text = "?"
+	## TODO: replace resolution setting with a different one
```

Comment:
supporting "resolution" setting for this kind of game seems unnecessary right now 

i'm open to ideas for another toggle setting here